apps/network-alerts/test/unit/notification/notification_test.js

```diff
@@ -1,15 +1,11 @@
 /* global MocksHelper,
   NotificationHandler,
-  Utils
  */
 'use strict';
 
-require('/test/unit/mock_utils.js');
 require('/js/notification/notification.js');
 
-var mocksForNotification = new MocksHelper([
-  'Utils'
-]);
+var mocksForNotification = new MocksHelper([]);
 
 suite('Network Alerts - Notification handling', function() {
   mocksForNotification.attachTestHelpers();
```

Comment:
It doesn't have any helpers to attach anymore :) So you can remove all ```mocksForNotification``` references entirely.